CMakeLists.txt

```diff
@@ -146,11 +150,8 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU"
         target_compile_options(xeus PUBLIC -march=native)
     endif()
 
-    CHECK_CXX_COMPILER_FLAG("-std=c++11" HAS_CPP11_FLAG)
-    if(HAS_CPP11_FLAG)
-        target_compile_options(xeus PUBLIC -std=c++11)
-    else()
-        message(FATAL_ERROR "Unsupported compiler -- xeus requires C++11 support!")
+    if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+        target_compile_options(xeus public -stdlib=libc++)
```

Comment:
We cannot do that unfortunately. On linux, libc++ does not work well with streams